srcpkgs/dunst/template

```diff
@@ -1,20 +1,19 @@
 # Template file for 'dunst'
 pkgname=dunst
-version=1.5.0
+version=1.6.1
 revision=1
 build_style=gnu-makefile
 make_use_env=yes
-make_build_target="all dunstify"
 hostmakedepends="perl pkg-config"
 makedepends="gtk+-devel libXScrnSaver-devel libXinerama-devel libXrandr-devel
- libxdg-basedir-devel libnotify-devel"
+ libxdg-basedir-devel libnotify-devel wayland-devel wayland-protocols"
```

Comment:
I'm about 100% sure `wayland-protocols` isn't making any difference here.

Reply:
Unless it's being used to determine if Wayland should be built at all.

Reply:
> WAYLAND=(0|1): Disable/Enable wayland support. (Default: 1 (enabled))

I think it always tries to build with Wayland.

Reply:
Package wayland-protocols was not found in the pkg-config search path.
Perhaps you should add the directory containing `wayland-protocols.pc'
to the PKG_CONFIG_PATH environment variable
No package 'wayland-protocols' found
Makefile:40: "Failed to query pkg-config for package 'wayland-protocols'!"

Yes enable build default, so you mean you would like wayland be put in a build_option_ ?